app/helpers/office_helper.rb

```diff
@@ -10,6 +12,12 @@ def office_address(office)
   end
 
   def display_website(office, firm)
-    office.website.present? ? office.website : firm.website_address
+    user_uri = office.website.present? ? office.website : firm.website_address
```

Comment:
what do you think about using the [presence method](https://apidock.com/rails/Object/presence)?. It would shorten it to:
`user_uri = office.website.presence || firm.website_address`

Reply:
Good point, absolutely 👍